content/20.pharmaceuticals.md

```diff
@@ -699,52 +699,88 @@ Additionally, different studies used different dosages, number of doses, and met
-Interestingly, the patient-isolated antibodies did not cross-react with RBDs from SARS-CoV-1 and MERS-CoV, although cross-reactivity to the trimeric spike proteins of SARS-CoV-1 and MERS-CoV was observed.
-This finding suggests that the RBDs between the three coronavirus species are immunologically distinct and that the isolated nAbs targeting the RBD of SARS-CoV-2 are species specific.
-While this specificity is desirable, it also raises the question of whether these antibodies are more susceptible to viral escape mechanisms.
-Viral escape is a common resistance mechanism to nAb therapy due to selective pressure from nAbs [@doi:10.2217/imt.15.33; @doi:10.1126/science.1213256].
-For HIV, broadly neutralizing antibodies (bnAbs) targeting the CD4 binding site (CD4bs) show greater neutralization breadth than monoclonal antibodies, which target only specific HIV strains [@doi:10.1016/j.molmed.2019.01.007].
-For MERS-CoV, a combination of multiple nAbs targeting different antigenic sites prevented neutralization escape [@doi:10.1128/JVI.02002-17].
-It was found that the different antibody isolates did not target the same epitopes, suggesting that using them in combination might produce a synergistic effect that prevents viral escape [@doi:10.1101/2020.03.21.990770].
-It was also demonstrated that binding affinity of the antibodies does not reflect their capability to compete with ACE2 binding.
-Furthermore, no conclusions about correlations between the severity of disease and the ability to produce nAbs can be drawn at this point.
-Rather, higher nAb titers were more frequently found in patients with severe disease.
-Correspondingly, higher levels of anti-spike IgG were observed in patients that died from infection compared to patient that recovered [@doi:10.1172/jci.insight.123158].
-
-Results from the SARS and MERS epidemics thus provide valuable lessons for the design of nAbs for the current outbreak.
-The findings for SARS-CoV-1 and MERS-CoV can aid in identifying which structures constitute suitable targets for nAbs, despite the fact that the RBD appears to be distinct between the three coronavirus species.
-These studies also suggest that a combination of nAbs targeting distinct antigens might be necessary to provide protection [@doi:10.1128/JVI.02002-17].
-The biggest challenge remains identifying antibodies that not only bind to their target, but also prove to be beneficial for disease management.
-On that note, a recently published study indicates that anti-spike antibodies could make the disease worse rather than eliminating the virus [@doi:10.1172/jci.insight.123158].
-These findings underscores our current lack of understanding the full immune response to SARS-CoV-2.
-<!-- TO DO : Confirm up to date-->
+#### Monoclonal Neutralizing Antibodies
+
+Monoclonal antibodies have revolutionized the way we treat human diseases. 
```

Comment:
```suggestion
Monoclonal antibodies, antibodies that recognize a single epitope, have revolutionized the way we treat human diseases. 
```

Reply:
Not sure if that is necessary. I see where you are coming from but I think people usually know this.

Reply:
Yea this was definitely coming someone like me that at one point knew the difference between mono and poly Ab but had to relook it up cuz it had been a while:) I'm fine removing it